.test/run.sh

```diff
@@ -608,6 +608,53 @@ fi
 	;&
 	password_hash)
 
+	echo -e "Test: Replica container can be initialized with environment variables\n"
+
+	mariadb_replication_user='foo'
+	mariadb_replication_password='foo123'
+	
+	runandwait -e MARIADB_REPLICATION_MODE="master" \
+		-e MARIADB_REPLICATION_USER="$mariadb_replication_user" \
+		-e MARIADB_REPLICATION_PASSWORD="$mariadb_replication_password" \
+		-e MARIADB_DATABASE=repcheck \
+		"$image"
+	
+	count_primary=$(mariadbclient -u $mariadb_replication_user -proger repcheck --batch --skip-column-names -e 'select sum(i) from t1')
```

Comment:
a) wrong password (not roger), no `replcheck` database or table created.

It doesn't appear that the user has been created. 

For this test, something simple for -e like `show grants` and ensure the right grants is the value would be good for a test.

Reply:
Lets get the test running before the next review :smiley:

Reply:
also useful for testing the entrypoint script:
```
$ docker run --name mariadbcontainer15397 --rm --publish 3306 -e MARIADB_REPLICATION_MODE=master -e MARIADB_REPLICATION_USER=foo -e MARIADB_REPLICATION_PASSWORD=foo123 -e MARIADB_ALLOW_EMPTY_ROOT_PASSWORD=1  -e MARIADB_DATABASE=repcheck --user mysql ms105 bash -x -v /usr/local/bin/docker-entrypoint.sh mariadbd
```